astroid/raw_building.py

```diff
@@ -471,7 +477,26 @@ def imported_member(self, node, member, name: str) -> bool:
             # check if it sounds valid and then add an import node, else use a
             # dummy node
             try:
-                getattr(sys.modules[modname], name)
+                with redirect_stderr(io.StringIO()) as stderr, redirect_stdout(
+                    io.StringIO()
+                ) as stdout:
+                    getattr(sys.modules[modname], name)
+                    stderr_value = stderr.getvalue()
+                    if stderr_value:
+                        logger.error(
+                            "Captured stderr while getting %s %s:\n%s",
+                            sys.modules[modname],
+                            name,
```

Comment:
```suggestion
                            "Captured stderr while getting %s from %s:\n%s",
                            name,
                            sys.modules[modname],
```
Does this make sense?